### PR TITLE
Updates to Readme and Changelog for version 0.12.0

### PR DESCRIPTION
**Pull request by zeibura**
- client readme file updated with same content as main file
- First draft added for the changelog (in the root file). If you agree with the content, I will copy it over to the client changelog before merging to dev.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,32 @@
...
+- Updates to README
+
+#### Fixed
+
+- Create a .gitignore file if it doesn't exist
+- Fix glob pattern for dynamic registration
+- Fix issue with indirect overwrite settings conf for DSN paths
+- Fix issues related to unrecognised copybook extensions
+- Fix representation of settings.json in Theia
+- Use sync APIs to create a folder and write to a file to avoid unexpected behavior
```
Comment (temanbrcom) [inline CHANGELOG.md:25]: Previously have been adding PR links, I think we should keep it.

Comment (zeibura) [reply CHANGELOG.md:25]: We were. I was told by @zimlu02 to stop doing this as of v0.11.0 and to use the added/fixed/changed format, not listing every PR or commit and just summarising main ones, without PR numbers.
We need to agree on a format for the changelog going forward, we can't just keep changing our minds about it. Let's talk about it at the standup today.

**Comment by zeibura** [comment]
OK I've updated the changelogs as suggested
The one in the root folder is the list of PRs (with duplicates removed - I think the ones with practically no description were duplicates)
The one in the client folder is the more compact one. Please check the contents and see if they are accurate / contain all necessary info.

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,31 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+## [0.12.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.11.1...0.12.0) (2020-07-07)
+
+#### Added
+
+- Support for local copybooks
+- Dynamic registration for watchers
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:9]: Maybe, we may drop this point? It is a part of the local copybooks support

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,31 @@
...
+- Updates to README
+
+#### Fixed
+
+- Create a .gitignore file if it doesn't exist
+- Fix glob pattern for dynamic registration
+- Fix issue with indirect overwrite settings conf for DSN paths
+- Fix issues related to unrecognised copybook extensions
+- Fix representation of settings.json in Theia
+- Use sync APIs to create a folder and write to a file to avoid unexpected behavior
```
Comment (temanbrcom) [inline clients/cobol-lsp-vscode-extension/CHANGELOG.md:27]: This is applicable only for test and doesn't affect the user experience, so we may drop it

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md (hunk 2 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md]: done in latest commit

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,31 @@
...
+#### Changed
+
+- Adjust the info message about missing copybooks
+- Reduce the number of error/informational messages
+- Inform user about wrong credentials and stop sending other MF requests
+- Copybooks downloaded from the mainframe are now downloaded to /.c4z/.copybooks in the workspace
+- Settings parameter for DSN paths changed
+- Updates to README
+
+#### Fixed
```
[the remark this answers is not in the document]
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md:19]: done in latest commit

File: clients/cobol-lsp-vscode-extension/CHANGELOG.md (hunk 3 above)
Comment (zeibura) [reply clients/cobol-lsp-vscode-extension/CHANGELOG.md]: done in latest commit

**Pull request merged**